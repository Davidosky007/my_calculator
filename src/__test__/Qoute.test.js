/* eslint-disable linebreak-style */
import { create } from 'react-test-renderer';
import '@testing-library/jest-dom';
import Qoutes from '../components/Qoute';

describe('test for Qoutes component', () => {
  test('renders Qoutes component properly', () => {
    const qoute = create(<Qoutes />).toJSON();
    expect(qoute).toMatchSnapshot();
  });
});
