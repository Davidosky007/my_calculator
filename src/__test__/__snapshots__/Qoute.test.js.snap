// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Qoutes component renders Qoutes component properly 1`] = `
<div
  className="main_qoute"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.  -William Paul Thurston
  </p>
</div>
`;
